requirements.txt

```diff
@@ -8,3 +8,5 @@ python-dotenv
 pydantic
 requests
 termcolor
+pytest
```

Comment:
Can we revert these here and add a `requirements.dev.txt` perhaps where these are added? The `llama-stack` package is used for some lightweight CLI commands (predominantly `llama model download`) and we don't want those folks to need unnecessary dependencies.

Reply:
done